Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -407,9 +407,9 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
       .UseStandardOperatorParameters("number");
 
   obj.AddAction("ModVarObjetTxt",
-                _("Text of an object variable"),
-                _("Change the text of an object variable."),
-                _("the text of variable _PARAM1_"),
+                _("String of an object variable"),
```

Comment:
in the related issue, I think there was a suggestion to change *all* actions & conditions descriptions to "String (Text)"